lib/firebase/notifications.dart

```diff
@@ -104,12 +105,29 @@ class Notifications {
 
   /// Retrieves and saves the FCM token for the device.
   Future<void> getFCMToken() async {
+    if (Platform.isIOS) {
+      await _firebaseMessaging.getAPNSToken();
+    }
+
     final fCMToken = await _firebaseMessaging.getToken();
     final user = await _authService.getUserData();
+
+    ZOLogger.logMessage('FCM token: $fCMToken', isError: true);
```

Comment:
Why error?

Reply:
I forgot it there. It was for testing purposes.